src/workerd/server/tests/container-client/test.js

```diff
@@ -46,15 +46,20 @@ export class DurableObjectExample extends DurableObject {
     // Test HTTP requests to container
     {
       let resp;
-      for (let i = 0; i < 6; i++) {
+      const maxRetries = 20;
+      for (let i = 1; i <= maxRetries; i++) {
         try {
           resp = await container
             .getTcpPort(8080)
             .fetch('http://foo/bar/baz', { method: 'POST', body: 'hello' });
           break;
         } catch (e) {
-          await scheduler.wait(500);
-          if (i === 5) {
+          console.info(
+            `Retrying getTcpPort() call due to an error: ${e.message}`
+          );
+          console.info(e);
```

Comment:
No need to log the error twice.

```suggestion
          console.info(
            `Retrying getTcpPort() call due to an error: ${e.stack}`
          );
```